apis/nucleus/src/components/listbox/components/ListBoxRowColumn/components/ListBoxRoot.jsx

```diff
@@ -68,7 +68,7 @@ const RowColRoot = styled('div', {
     minWidth: 0,
     flexGrow: 1,
     // Note that this padding is overridden when using checkboxes.
-    paddingLeft: `${CELL_PADDING_LEFT}px`,
```

Comment:
this (and the other changes) should not looks att isGridMode but if the current layout show Ticks (/Icons)

Reply:
Updated!